ci/Dockerfile

```diff
@@ -27,7 +27,6 @@ FROM gpuci/cuda:$CUDA_VERSION-devel-centos7
 RUN yum install -y centos-release-scl
 RUN yum install -y devtoolset-9 rh-python38 epel-release
 RUN yum install -y zlib-devel maven tar wget patch ninja-build
-RUN yum -y install https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm && yum install -y git
```

Comment:
the CI also required git to do some work, otherwise it will fail 
```bash
ci/premerge-build.sh: line 22: git: command not found
```

Reply:
seeing the original repo is invalid.
Our CI requires some newer git version instead of default yum pkg. Let me try find available one

Reply:
merged yum repo fix https://github.com/NVIDIA/spark-rapids-jni/pull/208. and also fixed internal pipeline to cover dockerfile change cases in this PR.

Please help upmerge your branch and try re-trigger the build, thanks~